### ISSUE TITLE
Add an option to roll d16 for dual wielding

### ISSUE THREAD

**Issue by algnc** [cyface/foundryvtt-dcc#52]
Players using 2handed weapons or Halflings dual wielding could use an option to roll a d16 for initiative or attack rolls. Ideally they could set the d16 for specific weapons.

**Comment by algnc** [comment]
1d14 for shield bash would be nice too

**Comment by mooped** [comment]
There's a Shield Bash button on the Dwarf Skills tab, but it's not a great solution since it's in a different place to the other attacks and it doesn't currently support adding the attack bonus.

Once my weapons and armor refactor is done I think the best way to do a shield bash will be as a weapon item with a custom action die.

Likewise dual wielded weapons will be able to be assigned a d16 attack die.

Hopefully this solves the issue, but let me know if you think there's a case that doesn't cover.

Regarding initiative I need to think about how best to fit it in. Ctrl + Click to modify it as suggested in another issue would work but isn't particularly discoverable. A second '2H' label next to initiative might work if it doesn't feel too cluttered.

I'd rather avoid iterating the weapons and checking for 2 handed weapons automatically as it relies on players to keep their active weapon flagged up properly at all times which in my experience might cause more confusion.

**Comment by mooped** [comment]
Splitting this info a separate issue for two handed initiative
